manifests/0000_30_18-operatorstatus.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: operatorstatus.openshift.io
+kind: ClusterOperator
+metadata:
+  namespace: openshift-operator-lifecycle-manager
+  name: operator-lifecycle-manager
+status:
+  version: 1.0.0
```

Comment:
Should the version match the latest release manifest (looks like 0.8.0 currently)?

Reply:
Hm, not sure here either. OpenShift tracks master so I think the best thing would be do the git sha if possible.

Reply:
Would modifying "make release" be appropriate for replacing the SHA before building an image or is there somewhere else I should be looking?